src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -452,8 +451,8 @@ export class DynamicFacet extends Component implements IAutoLayoutAdjustableInsi
     this.dynamicFacetQueryController.enableFreezeFacetOrderFlag();
   }
 
-  public pinFacetPosition() {
-    this.padding && this.padding.pin();
+  public scrollToTop() {
+    ResultListUtils.scrollToTop(this.root);
```

Comment:
Just as an escape hatch, it should be possible to configure an option `enable-scroll-to-top="false"` for facet(s), because it might not work properly in all cases/setup/scenarios.

Clients might put the facet in some kind of dropdown, I've even seen clients use hidden facet to retrieve information to be used elsewhere in the page, so we cannot always assume that a facet is a "visible"/"standard" widget in the page that end user interact with.

This also should only execute if it's an actual end user action.
What if there's code that select values programmatically ? 
It should not scroll back up in that case, because we don't know what caused the selection.

Only in the case of a end user clicking on a facet value do we want this to happen.

Reply:
Sure, good idea to add that option.

> This also should only execute if it's an actual end user action.
That's already the case. On a facet select/deselect/clear user interaction (mouse/keyboard)

Reply:
For the second part of my comment (which you quoted), what does that mean ? :D 

Is it already the case ?